tests/Unit/Subscriber/Cache/PurgeActionsSubscriber/TestPurgeUserCache.php

```diff
@@ -0,0 +1,80 @@
+<?php
+namespace WP_Rocket\Tests\Unit\Subscriber\Cache\PurgeActionsSubscriber;
+
+use WP_Rocket\Subscriber\Cache\PurgeActionsSubscriber;
+use WP_Rocket\Tests\Unit\TestCase;
+use Brain\Monkey\Functions;
+use Brain\Monkey\Filters;
+
+/**
+ * @coversDefaultClass \WP_Rocket\Subscriber\Cache\PurgeActionsSubscriber
+ */
```

Comment:
Add the `@group` tag here to apply it to all test methods in this class, ie instead of having it on each method.